tests/conditional_compilation/test_infer_tool.py

```diff
@@ -0,0 +1,58 @@
+import argparse
+import logging as log
+import sys
+
+log.basicConfig(format="[ %(levelname)s ] %(message)s", level=log.INFO, stream=sys.stdout)
+import os
+
+import numpy as np
+from openvino.inference_engine import IENetwork
+
+
+def python_api_infer(ir_path, device):
```

Comment:
What is the reason for `python_api` in a function name? Should be name it `infer` or `do_infer`?